packages/flutter/lib/src/material/tabs.dart

```diff
@@ -844,6 +844,12 @@ class _TabBarScrollController extends ScrollController {
   }
 }
 
+/// Signature for [TabBar] callbacks that report that an underlying value has
+/// changed for a given [Tab] at `index`.
```

Comment:
maybe give an example for what `value` might be for each case. e.g. for `onHover` the value means hovered

Reply:
Added some more docs! :)